utils/helpers/keys.js

```diff
@@ -0,0 +1,58 @@
+import { proto } from "@hashgraph/proto";
+
+import mirrorNodeClient from "../../mirrorNodeClient.js";
+import consensusInfoClient from "../../consensusInfoClient.js";
+import { keyTypeConvertFunctions } from "./constants/key-type.js";
+
+/**
+ * Retrieves the encoded hexadecimal representation of a specified dynamic key
+ * from an account's information in the consensus layer.
+ *
+ * @async
+ * @param {string} accountId - The ID of the account from which to retrieve the key.
+ * @param {string} keyName - The name of the dynamic key to retrieve (e.g., wipeKey, freezeKey).
+ * @returns {Promise<string>} - A promise that resolves to the hexadecimal representation of the encoded key.
+ */
+export const getEncodedKeyHexFromKeyListConsensus = async (
+  accountId,
+  keyName,
```

Comment:
What it this param for? It's always assigned to "key" - I don't see other key from the account object from the mirror node swagger.

Reply:
I have refactored the helper functions to work with AccountTransaction, TokenTransaction, and similar types. See the new version, sorry about that delay.